contracts/ElectionPractice.sol

```diff
@@ -0,0 +1,166 @@
+pragma solidity ^0.5.0;
+
+/**
+ * @title - How to use this?
+ * @dev - deploy contract to DEXON netowkr
+ * @dev - copy deployed address to https://dexon-workshop-vote.netlify.com/
+*/
+
+/**
+ * @title - In this Election Contract, it could take as many rounds of election.
+ *        - As a user of this contract, you are able to
+ *        - 1. Register as a candiate of the election.
+ *        - 2. One vote the candiate of your choose, and can fund any of the candiate as much money as you want.
+ *        - 3. Candiate will get the fund if the round is over.
```

Comment:
Candidates will be refunded when the current round is over

Reply:
ok